resources/js/osu-core.ts

```diff
@@ -89,6 +93,8 @@ export default class OsuCore {
       document.addEventListener('DOMContentLoaded', this.updateCurrentUser);
     }
     $.subscribe('user:update', this.onCurrentUserUpdate);
+    // ensure currentUser is updated early enough.
```

Comment:
I kinda doubt it's still relevant especially with `main.coffee` (where it was before) being loaded after this file